framework_tvb/tvb/core/services/operation_service.py

```diff
@@ -303,6 +304,53 @@ def load_view_model(self, adapter_instance, operation):
             h5_file.load_into(view_model)
         return view_model
 
+    def _review_operation_inputs(self, operation_gid):
+        """
+        :returns: A list of DataTypes that are used as input parameters for the specified operation.
+                 And a dictionary will all operation parameters different then the default ones.
+        """
+        operation = dao.get_operation_by_gid(operation_gid)
+        parameters = json.loads(operation.parameters)
+        try:
+            adapter = ABCAdapter.build_adapter(operation.algorithm)
+            return self.review_operation_inputs(adapter, operation)
+
+        except Exception:
+            self.logger.exception("Could not load details for operation %s" % operation_gid)
```

Comment:
When exactly is this exception happening?
I suspect is when importing old operations (e.g. DefaultProject in previous forms). Is this correct?
If so, then pls adjust logged messages accordingly.